package/Makefile

```diff
@@ -45,12 +45,18 @@ CLEAN_FLIGHT := YES
 endif
 
 # Set up targets
-ALL_BOARDS        := coptercontrol pipxtreme revomini sparky sparky2 sparkybgc quanton colibri flyingf3 flyingf4
+ALL_BOARDS        := coptercontrol pipxtreme revomini sparky sparky2 sparkybgc quanton colibri flyingf3 flyingf4 naze32
```

Comment:
at some point we should split all these into something that is auto-detected based on what is present in the targets directory. @solidgoldbomb if you have a brilliant makefile -foo this is one of those places that ends up creating merge conflicts when we add/remove targets